labonneboite/common/models/office.py

```diff
@@ -115,20 +118,39 @@ class Office(FinalOfficeMixin, CRUDMixin, Base):
     def __unicode__(self):
         return u"%s - %s" % (self.siret, self.name)
 
-    def as_json(self, rome_code=None, distance=None, zipcode=None, extra_query_string=None):
+    def as_json(self,
+            rome_codes=None,
+            hiring_type=hiring_type_util.DEFAULT,
```

Comment:
Pour des raisons d'homogénéité des arguments (et par simplicité) je propose de faire que la valeur par défaut de `hiring_type` soit None.

Reply:
done